portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,47 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.Date;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+ArrayList<String> commentList = new ArrayList<String>();
```

Comment:
Declare the type as just List<String>. Use ArrayList<> only for instantiation.

Reply:
When I tried to do that, I got an error that said that I couldn't convert a list into an arraylist